de.tubs.variantsync.core/src/de/tubs/variantsync/core/managers/data/FeatureContext.java

```diff
@@ -25,6 +25,7 @@ public FeatureContext(String name, FeatureColor highlighter) {
 	}
 
 	public boolean isComposed() {
+		// if the name contains an or, and or not in its middle, the FeatureContext is supposed to be composed
```

Comment:
please put the operator names into the quotation marks as it is hard to read otherwise.